src/components/Nav.svelte

```diff
@@ -1,12 +1,13 @@
 <script>
     import { fade, slide } from 'svelte/transition';
     import Hamburger from 'svelte-hamburgers';
-    import { isActive } from '@roxi/routify';
+    import { isActive, isChangingPage } from '@roxi/routify';
 
     let width;
     let open;
 
     $: mobile = width < 900;
+    $: if (!mobile || $isChangingPage) open = false;
```

Comment:
What's the `isChangingPage` for?

Reply:
So this will automatically close the nav when a new link is clicked.  Otherwise you would need to add `on:click={() => open = false}` to each `a` tag.

Reply:
Oh shit that's super clever!